CompetitionTemplate/src/competition/subsystems/arm/ArmSubsystem.java

```diff
@@ -16,7 +16,8 @@
 public class ArmSubsystem extends BaseSubsystem {
 
     private static Logger log = Logger.getLogger(ArmSubsystem.class);
-    public XSpeedController armMotor;
+    public XSpeedController armMotor1;
```

Comment:
lets name these left and right so when electrical needs to invert one we know which we're talking about.